lib/ansible/galaxy/role.py

```diff
@@ -379,7 +379,11 @@ def install(self):
                                     if n_part != '..' and not n_part.startswith('~') and '$' not in n_part:
                                         n_final_parts.append(n_part)
                                 member.name = os.path.join(*n_final_parts)
-                                role_tar_file.extract(member, to_native(self.path))
+                                # deprecated: description='extract fallback without filter' python_version='3.11'
+                                if hasattr(tarfile, 'fully_trusted_filter'):
```

Comment:
Is there a reason this needs to be `fully_trusted_filter`?

It seems like this could use `data_filter` just like the other cases.

Reply:
Looking at description, I think `fully_trusted_filter` would work for all except tests (which is marked as `data_filter`). I might be wrong.

Reply:
We should use `data_filter` in all cases unless there's a known reason why it will disallow something we want.